health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/StackedTable.js

```
@@ -0,0 +1,237 @@
+          return data;
+        }
+      });
+    }
+
+    if (filteredData?.length) {
+      return filteredData?.map((item) => StackedRowV2(item));
+    } else {
+      return <div className="digit-stacked-row">{t("DSS_NO_RESULTS")}</div>;
+    }
+  };
+  const subHead = "SUB_" + visualizer?.name;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)